compiler/rustc_borrowck/src/diagnostics/mod.rs

```diff
@@ -623,6 +623,28 @@ impl UseSpans<'_> {
         }
     }
 
+    // callback a subdiag on the label-span
+    pub(super) fn var_span_subdiag_label(
```

Comment:
```suggestion
    pub(super) fn var_subdiag(
```
This is a little cryptic but maybe more accurate?